docs/reference/commandline/volume_rm.md

```diff
@@ -0,0 +1,24 @@
+<!--[metadata]>
++++
+title = "ps"
+description = "the volume rm command description and usage"
+keywords = ["volume, rm"]
+[menu.main]
+parent = "smn_cli"
+weight=1
```

Comment:
remove the weight.  No need for the heading or the bold:

--->

<!--[metadata]>
+++
title = "ps"
description = "the volume rm command description and usage"
keywords = ["volume, rm"]
[menu.main]
parent = "smn_cli"
+++
<![end-metadata]-->

# volume rm

    Usage: docker volume rm [OPTIONS] [VOLUME NAME]

    Remove a volume

    --help=false       Print usage

Removes a volume. You cannot remove a volume that is in use by a container.

  $ docker volume rm hello